sdk/keyvault/azure-keyvault-certificates/tests/test_examples_certificates.py

```diff
@@ -54,11 +54,23 @@ def test_example_certificate_crud_operations(self, vault_client, **kwargs):
         certificate_operation_poller = certificate_client.create_certificate(name=cert_name, policy=cert_policy)
 
         # Here we are waiting for the certificate creation operation to be completed
-        certificate_operation_poller.wait()
+        certificate = certificate_operation_poller.result()
 
         # You can get the final status of the certificate operation poller using .result()
         print(certificate_operation_poller.result())
 
+        print(certificate.id)
+        print(certificate.name)
+        print(certificate.policy.key_properties.exportable)
+        print(certificate.policy.key_properties.key_type)
+        print(certificate.policy.key_properties.key_size)
+        print(certificate.policy.key_properties.reuse_key)
+        print(certificate.policy.content_type)
+        print(certificate.policy.issuer_name)
+        print(certificate.policy.subject_name)
+        print(certificate.policy.san_dns_names)
+        print(certificate.policy.validity_in_months)
```

Comment:
This makes a long doc snippet. Do we need to show all these properties?

Reply:
let me know what you think about the properties I ended up including in the snippet, thanks!